pkg/capabilities/datastreams/types.go

```diff
@@ -65,10 +66,16 @@ type FeedReport struct {
 	ObservationTimestamp int64
 }
 
+// passed alongside Streams trigger events
+type SignersMetadata struct {
+	Signers               [][]byte
+	MinRequiredSignatures int
+}
+
 //go:generate mockery --quiet --name ReportCodec --output ./mocks/ --case=underscore
 type ReportCodec interface {
-	// validate each report and convert to a list of Mercury reports
-	Unwrap(raw values.Value) ([]FeedReport, error)
+	// validate each report and convert to a list of Feed reports
+	Unwrap(wrapped values.Value, allowedSigners [][]byte, minRequiredSignatures int) ([]FeedReport, error)
```

Comment:
this doesn't look like Unwrap any longer. UnwrapAndValidate? FeedsFromValue()? CreateFeeds()?

Reply:
How about UnwrapValid()? :)